components/new-contribution-flow/StepDetails.js

```diff
@@ -21,7 +21,7 @@ import TierCustomFields from './TierCustomFields';
 
 const StepDetails = ({ onChange, data, collective, tier, showFeesOnTop, isTaxDeductibleInTheUS }) => {
   const intl = useIntl();
-  const presets = React.useMemo(() => getTierPresets(tier, collective.type), [tier, collective.type]);
+  const presets = React.useMemo(() => getTierPresets(tier, collective?.type), [tier, collective?.type]);
```

Comment:
The tier is optional, but not the collective. If it's missing we need to investigate why.

```suggestion
  const presets = React.useMemo(() => getTierPresets(tier, collective.type), [tier, collective.type]);
```